nzsmd-flatfiles/explanation_of_columns.md

```diff
@@ -0,0 +1,81 @@
+## Explanation of columns in flatfiles
+
+| Column name  | Description   |
+| ------------- | ------------- |
+| CuspID | GeoNet event identification number |
+| Origin time    | Origin time of earthquake in UTC  |
+| Record | Record name |
+| Mw | Moment magnitude |
+| MwUncert | Uncertainty in moment magnitude estimate |
+| TectClass | Tectonic classification, either 'crustal', 'interface' or 'slab' |
+| Mech | Focal mechanism, as defined by McVerry et al., (2006)
+| | S → strike-slip, with rake angle δ, −33 ≤ δ ≤ 33, 147 ≤ δ ≤ 180, −180 ≤ δ ≤ −147 |
+| | N → normal, −146 ≤ δ ≤ −34 |
+| | R → reverse, 67 ≤ δ ≤ 123 |
+| | O → oblique with a reverse component, 34 ≤ δ ≤ 66, 124 ≤ δ ≤ 146 |
+| | U → unknown |
+| PreferredFaultPlane | 1 if one fault plane orientation is preferred out of the two conjugate fault planes in the moment tensor |
+| | 0 if the preferred fault plane is unknown |
+| Strike | Strike angle (degrees)|
+| | If PreferredFaultPlane = 1, this is the likely fault strike, otherwise it is only one of two possible strikes |
+| Dip | Dip angle (degrees) |
+| Rake | Rake angle (degrees) |
+| Location | Reference for the preferred location |
+| | special → special studies (see references)
+| | geonet → standard GeoNet catalogue |
+| | nlloc → relocation using the NonLinLoc algorithm |
+| | simulps → relocation using the SimulPS algorithm |
+| | fdsn → the FDSN catalogue |
```

Comment:
Which agency supplies this FDSN catalogue? USGS? FDSN is a webservice that a lot of different agencies provide including us.